util/fee-estimator/src/error.rs

```diff
@@ -0,0 +1,20 @@
+//! The error type for the fee estimator.
+
+use thiserror::Error;
+
+/// A list specifying general categories of fee estimator errors.
+#[derive(Error, Debug, PartialEq)]
+pub enum Error {
+    /// Dummy fee estimator is used.
+    #[error("dummy fee estimator is used")]
+    Dummy,
+    /// Not ready for do estimate.
+    #[error("not ready")]
```

Comment:
How about given a more verbose errro: `#[error("not ready, node is in IBD mode")]`